official/recommendation/ncf_keras_main.py

```diff
@@ -155,13 +157,13 @@ def _get_keras_model(params):
   # is designed to be of batch_size 1 for each replica.
   user_input = tf.keras.layers.Input(
       shape=(batch_size,),
-      batch_size=1,
+      batch_size=params["batches_per_step"],
```

Comment:
hm, why is this change in this PR? Or am i seeing a bad merge result or something?

Reply:
No, it is because of cl/243685134. Now we are supposed to have global batch_size in the input layers.